test/parallel_api/ranges/reverse_ranges_sycl.pass.cpp

```diff
@@ -37,7 +37,7 @@ main()
     sycl::buffer<int> A(max_n);
 
     auto iota = views::iota(0, max_n);
-    ::nano::ranges::copy(iota, views::host_all(A).begin());
```

Comment:
Here and in other tests, why do we refer to `namespace nano` at all, instead of our own namespace where all the same names should have been injected?

Reply:
The special comments were added in the code of these tests